lib/rspec/core/option_parser.rb

```diff
@@ -226,6 +226,11 @@ def parser(options)
           (options[:full_description] ||= []) << Regexp.compile(Regexp.escape(o))
         end
 
+        parser.on('-E', '--example-matches STRING', "Run examples whose full nested names match REGEX (may be",
```

Comment:
Should probably say REGEX or PATTERN rather than string.

Reply:
@JonRowe thanks - not exactly sure what comment to add - can you be more explicit?  Or is it even something you can do yourself - you can edit our PR directly in github right?

Reply:
Hm thats a new feature :P, I've fixed the rubocop failures but your scenario isn't working!

Reply:
thanks - it seems to be passing on some versions of ruby and not others - do we need to test locally on all the different versions of ruby going back to 1.8.7?

Reply:
It needs to work on all versions back to 1.8.7 at this point in time, I would suggest seeing what regex or command line differences there maybe in older versions?

Reply:
👍